lib/CallBackery/GuiPlugin/AbstractTable.pm

```diff
@@ -144,14 +144,25 @@ sub makeExportAction {
                 firstRow => 0,
                 lastRow => $self->getTableRowCount($args)
             });
-            my @titles = map {$_->{key}} @{$self->tableCfg};
+
+            # Use the translated table headers in row 1.
+            # Or the keys if undefined.
+            my $loc = CallBackery::Translate->new(localeRoot=>$self->app->home->child("share"));
+            $loc->setLocale($self->user->userInfo->{lang});
```

Comment:
{lang} may be undef ... what happens in that case ?

Reply:
it does not translate and gives an "uninit value" etc.